src/Type/IntegerType.php

```diff
@@ -83,6 +84,7 @@ public function toString(): Type
 		return new IntersectionType([
 			new StringType(),
 			new AccessoryLowercaseStringType(),
+			new AccessoryUppercaseStringType(),
```

Comment:
FloatType can be improved as well - https://github.com/phpstan/phpstan-src/pull/3613#discussion_r1835336748

Reply:
done